PyEveLiveDPS/fleetConnectionWindow.py

```diff
@@ -39,66 +44,110 @@ def __init__(self, mainWindow):
                 pass
         self.geometry("375x310")
         self.update_idletasks()
-        
+
         characterFrame = tk.Frame(self)
-        tk.Label(characterFrame, text="Character to use for fleet mode: ").grid(row="0", column="1", sticky="w")
+        tk.Label(characterFrame,
+                 text=tr("Character to use for fleet mode: ")).grid(row="0",
+                                                                    column="1",
+                                                                    sticky="w")
         nameLabel = tk.Label(characterFrame, text=self.characterName)
         font = tkFont.Font(font=nameLabel['font'])
         font.config(weight='bold')
         nameLabel['font'] = font
         nameLabel.grid(row="0", column="2", sticky="w")
-        nameDescription = tk.Label(self, text="To use a different character for fleet mode, choose a different\n" +
-                            " character in the 'Character...' menu on the main window")
+        nameDescription = tk.Label(
+            self,
+            text=tr(
+                "To use a different character for fleet mode, choose a different\n"
+            ) + tr(" character in the 'Character...' menu on the main window"))

```

Comment:
Should this be a single translation string?

Reply:
yes, I think i used "global replace" to wrap the strings. If you need this i can fix it.